### DISCUSSION TITLE
验证服务器炸了吗

### DISCUSSION THREAD

**Discussion by sajfasjg81** [Mrs4s/go-cqhttp#2137] [Q&A]
![image](https://github.com/Mrs4s/go-cqhttp/assets/132623805/6d1bb2d9-3ad2-425e-baf8-d8c4d45adb4c)

验证服务器炸了吗

**Comment by yyx2684264** [comment]
建议更新最新版本试试，在V1.0.0版本的changlog里面有写如下内容：

`[v1.0.0](https://github.com/Mrs4s/go-cqhttp/releases/tag/v1.0.0)
Changelog
本版本主要是修复错误.
本次更新后登录将不在依赖 captcha.go-cqhttp.org 域名. (大部分情况下)`

**Reply by Justworst** [reply]
同问，我也是这样的

**Comment by wuko233** [comment]
现在验证地址能正常访问了，但是验证完后程序无响应，启动时有这个报错：[2023-05-13 17:19:23] [ERROR]: 获取T544时出现问题: Post "https://captcha.go-cqhttp.org/sdk/dandelion/energy": context deadline exceeded (Client.Timeout exceeded while awaiting headers) ，感觉还是有问题
